### PR TITLE
[2.1.x] EOSIO GSG high-level workflow

### PR DESCRIPTION
**Pull request by bobgt**
Embedding a GSG high-level workflow diagram that illustrates the getting started experience.

### CODE REVIEW

**Review by iamveritas — APPROVED** [review]

File: docs/30_getting-started-guide/index.md [hunk 1]
```diff
@@ -3,12 +3,27 @@ content_title: "Getting Started Guide"
 link_text: "Getting Started Guide"
 ---
 
-The EOSIO _Getting Started Guide_ provides guided instructions on how to set up your local development environment. These instructions installs the EOSIO software components and other related dependencies for the EOSIO platform. This setup is an efficient and convenient method to onboard the EOSIO development ecosystem. Subsequently, you become acquainted with the smart contracts development workflow in a local environment.
+The purpose of the _EOSIO Getting Started Guide_ is to provide guided instructions to set up your local EOSIO development environment and use the environment to build and deploy a basic **_Hello World_** Smart Contract. The _Getting Started Guide_ high-level workflow is illustrated in the following diagram:
```
Comment (lparisc) [inline docs/30_getting-started-guide/index.md:6]: Suggestion:
development environment and use the environment --> development environment and use it

**Review by halsaphi — APPROVED** [review]
Same suggestions as PR 422

File: docs/30_getting-started-guide/index.md [hunk 2]
```diff
@@ -3,12 +3,27 @@ content_title: "Getting Started Guide"
 link_text: "Getting Started Guide"
 ---
 
-The EOSIO _Getting Started Guide_ provides guided instructions on how to set up your local development environment. These instructions installs the EOSIO software components and other related dependencies for the EOSIO platform. This setup is an efficient and convenient method to onboard the EOSIO development ecosystem. Subsequently, you become acquainted with the smart contracts development workflow in a local environment.
+The purpose of the _EOSIO Getting Started Guide_ is to provide guided instructions to set up your local EOSIO development environment and use the environment to build and deploy a basic **_Hello World_** Smart Contract. The _Getting Started Guide_ high-level workflow is illustrated in the following diagram: 
+
+![EOSIO Getting Started Workflow](../eosio-gsg-workflow.png)
```
Comment (lparisc) [inline docs/30_getting-started-guide/index.md]: This link is broken. Should be:
!\[EOSIO Getting Started Workflow\](eosio-gsg-workflow.png)

File: docs/30_getting-started-guide/index.md [hunk 3]
```diff
@@ -3,12 +3,27 @@ content_title: "Getting Started Guide"
...
+
+The diagram identifies the following two workflows: 
+
+1. **Set up local development environment**
+2. **Build and deploy a basic Hello World smart contract**
+
+[[info | Note]]
+| Before you begin with the getting started workflow, make sure you familiarize yourself with [EOSIO Platform and Toolchain](../20_introduction-to-eosio/10_platform_and_toolchain.md), [Core Concepts](../20_introduction-to-eosio/20_core_concepts.md), and some of the [Technical Features](20_introduction-to-eosio/30_technical_features.md).
```
Comment (lparisc) [inline docs/30_getting-started-guide/index.md]: The last link is broken. Should be:
\[Technical Features\](../20_introduction-to-eosio/30_technical_features.md)

**Review by lparisc — CHANGES_REQUESTED** [review]
Looks good, just fix the broken links.

File: docs/30_getting-started-guide/index.md (hunk 1 above)
Comment (bobgt) [reply docs/30_getting-started-guide/index.md:6]: I tried to be explicit by mentioning "and use the environment..."

The "it" may refer to the "getting started guide" or to the "environment." So to be clear I used "use the environment..."

**Review by lparisc — APPROVED** [review]

**Pull request merged**